gini/gini-vision-lib-ios
# Tip popover for document import button

When the app is opened for the first time, it shows a tip next to the file import button. It shouldn't appear again once it has been shown.

### How to test
Open the example app, go to the camera screen (either Screen or Component API) and see that the tip appears next to file import button. If you open the camera screen again you won't see it.

### Merging
Automatic.

### Ticket 
Issue #72
